src/man/sssd-ldap.5.xml

```diff
@@ -913,9 +913,37 @@
                             Specifies that the id_provider connection must also
                             use <systemitem class="protocol">tls</systemitem> to protect the channel.
                         </para>
+                        <para>
+                            Three modes are currently supported:
+                            <itemizedlist>
+                                <listitem>
+                                    <para>
+                                        allow - Attempt startTLS first then fallback
+                                        to connecting without if it fails.
+                                    </para>
+                                </listitem>
+                                <listitem>
+                                    <para>
+                                        true - TLS must be used or the connection
+                                        will fail.
+                                    </para>
+                                </listitem>
+                                <listitem>
+                                    <para>
+                                        false - startTLS will not be attempted, Not
+                                        recommended for security reasons.
+                                    </para>
+                                </listitem>
+                            </itemizedlist>
+                        </para>
                         <para>
                             Default: false
                         </para>
+                        <para>
+                            Used as a default for compatibility reasons, please
```

Comment:
How did this sentence got into this patch? It's absent in original 93717284b23ee14f9542fedc5f67f33a24d4eb10

Probably it should be included into 'ldap: Switch ldap_id_use_start_tls default to allow' patch?